### ISSUE TITLE
Expect to see channel closings in `chain-receives`

### ISSUE THREAD

**Issue by TKone7** [alexbosworth/balanceofsatoshis#130]
Version: 
10.12.0

When I run `$ bos accounting chain-receives` I would expect to see all transactions that increase my on-chain balance. However, I cannot see any returning funds from channel closures. According to your [guide](https://yalls.org/articles/97d67df1-d721-417d-a6c0-11d793739be9:0965AC5E-56CD-4870-9041-E69616660E6F/bc71e6bf-f2aa-4bae-a3e8-b12e7be2284c) this should be displayed. Is that correct?

**Comment by alexbosworth** [comment]
It was adjusted recently to omit funds moving out from yourself to yourself

There are some limits to what it can do because there is currently no API to determine if an address is owned by the backing LND wallet

The funds that are meant to appear from channel closures should really be ones where you are really receiving funds in the close not just changing the output constriction of funds you already had

**Comment by TKone7** [comment]
I understand. So the `accounting` should give an overall view of the node and not report transactions which are happening between on- and off-chain (of the same wallet), is that right? Should then not the same rules apply for `chain-sends`? There the funding of channels is also displayed.

**Comment by alexbosworth** [comment]
Yeah I think in chain sends the channel funding should be removed, provided there is no push amount

**Comment by TKone7** [comment]
I like this approach. Specially as from an accounting point of view it really does not matter whether the funds are sitting on-chain or in a channel. It might be tricky to apply this strictly with loop ins/outs.
